sklearn/titanic/README.md

```diff
@@ -0,0 +1,99 @@
+```
+gcloud compute images list
+```
+
+##### 2. Accessing and Configuring the Instance
+
+Accessing the newly created image is quite simple:
+```
+gcloud compute ssh MYINSTANCE
+```
+
+Once we ssh to the instance, we will need to use *pip* to install scikit-learn and some other packages:
+```
+# Run on the new instance
+sudo apt-get install python-pip
+pip install numpy pandas sklearn scipy tensorflow
+```
+We are not using tensorflow to train the model in this sample. However we will be using *gfile*, a python class implemented in tensorflow which unifies how we access the local and remote files.
+
+If you want to use GCS to store the dataset, the new instance will then require to obtain user access credentials. The following command will guide you through the process:
+```
+# Run on the new instance
+gcloud auth application-default login
+```
+
+##### 3. Running the code
+We need to make the dataset available to the training code. We will show two ways for the code to access the dataset.
+
+###### Copying it to the instance
+We can copy the dataset and use it directly inside the instance and read it as a local file. Fortunately, *gcloud* makes this an easy step:
+```
+gcloud compute scp ./train.csv  MYINSTANCE:~/
+```
+
+Note that the same method can be used to copy any other file (e.g. the python code).
+
+We can then run our python code in the instance to create a model:
+```
+# Run on the new instance
+python titanic.py --titanic-data-path ./train.csv --model-output-path ./model.pkl
+```
+
+which will generate the model and saves it on the instance.
+
+###### Reading it from Google Cloud Storage
+Another solution is to read the dataset directly from a Bucket in GCS. First, we need to create a Bucket. We will use *gsutil* to do it:
+```
+gsutil mb MYBUCKET
+gsutil cp ./train.csv  gs://MYBUCKET/MYFOLDER/train.csv
+```
+
+The two commands above will create a Bucket in GCS, and then copy the dataset into it.
+
+The advantage of using *gfile* in our python code is that we can use the exact same code and read the dataset from GCS:
+```
+# Run on the new instance
+python titanic.py --titanic-data-path gs://MYBUCKET/MYFOLDER/train.csv --model-output-path gs://MYBUCKET/MYFOLDER/model.pkl
+```
+
+which will create the model and stores it in the same location as our dataset in GCS.
+
+That is it!! We have successfully trained a classifier using scikit-learn on a GCE instance.
+
+## What is Next
+Depending on your needs, here are a few things you may consider:
+* You may require more resources (more CPU's, more RAM) to train your model, in which case your instance should have a different type.
+* You may also want another OS for your instance, which requires you to create it using a different image.
+* It may be a good idea to stop or even delete your instance after your model is trained to minimize the cost.
```

Comment:
As a separate bullet point, I would also suggest telling users about preemptible instances here.

Reply:
Should there be a clean up step above. 
I could see myself skipping this section.